src/main/kotlin/game/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,18 @@
+package game.racingcar.domain
+
+import game.racingcar.strategy.MoveStrategy
+
+class Car {
+    var blackBox = arrayListOf<Int>()
```

Comment:
<img width="631" alt="스크린샷 2022-05-15 오후 5 20 38" src="https://user-images.githubusercontent.com/50702723/168463754-bd678d7a-0772-4116-97a1-252556e2fd99.png">
<img width="611" alt="스크린샷 2022-05-15 오후 5 20 57" src="https://user-images.githubusercontent.com/50702723/168463759-f3a622ab-9e4b-4eb6-958e-e1718d52aa9d.png">


 가변성을 위하여 arrayListOf()를 사용했다면, 가변성에 대해 더 명확하게 표현해줄 수 있는 mutableListOf() 를 사용하는 방법도 있습니다. 두 리스트 모두 ArrayList로 이루어져 있습니다 😊


하지만, 가변성이 있는 list를 외부에다가 노출해버리면, 외부에서 값을 수정할 수가 있습니다.

```suggestion
    private val _blackBox: MutableList<Int> = mutableListOf()

    val blackBox: List<Int>
        get() = _blackBox.toList()
```

이럴 경우에는 backing properties를 사용하여 안에서는 가변을 지킬 수 있고, 바깥에서는 불변을 지킬 수가 있습니다.

참고 
- [backing_properties](https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties)